### ISSUE TITLE
com.helpscout.beacon.internal.presentation.ui.article.ArticleActivity Only fullscreen opaque activities can request orientation

### ISSUE THREAD

**Issue by SMJ93** [helpscout/beacon-android-sdk-sample#133]
* Beacon version: 2.2.0
* Android OS version: 8.0.0
* Device make/model: Galaxy A3(2017)

**Steps to reproduce**

We are seeing the following error in Crashlytics from a few users on Android OS 8.0.0:

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.tally.android/com.helpscout.beacon.internal.presentation.ui.article.ArticleActivity}: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2957)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3032)
       at android.app.ActivityThread.-wrap11()
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1696)
       at android.os.Handler.dispatchMessage(Handler.java:105)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6944)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
```

**Actual outcome**
Beacon crashes

**Expected outcome**
Beacon should not crash

**Comment by aliripe** [comment]
Hi @SMJ93,

Thank you for reporting this! I've been investigating and it looks like it's an issue that Google introduced in Android 8.0 and fixed in 8.1, for more info make sure to check out [this link](https://issuetracker.google.com/issues/68454482).

We're not restricting the orientation for any of our Activities in out SDK so is it possible that your application is?

I've logged a ticket internally to make sure that we change the style for that specific version so that it doesn't crash anymore.

**Ticket**: [ANDSDK-581](https://helpscout.atlassian.net/browse/ANDSDK-581)

**Comment by SMJ93** [comment]
Hi @aliripe,

Great! Thanks for the quick response.

**Comment by aliripe** [comment]
Hi @SMJ93, we've included a fix for this as part of our Beacon 2.2.2 release, do let us know if the problem persists. 
See the [changelog](https://github.com/helpscout/beacon-android-sdk-sample/blob/master/CHANGELOG.md) for full details.
